src/modules/imageresizer/dll/trace.cpp

```diff
@@ -0,0 +1,49 @@
+#include "stdafx.h"
+#include "trace.h"
+

```

Comment:
Could this file be made more generic, and applied to mutliple powertoys?

Reply:
Ryan,
not that I know, the telemetry APIs require literals, so passing variables to TraceLoggingWrite doesn't work, so each powertoy has to have its own file.